package/com.unity.formats.usd/Editor/Recorder/UsdRecorderSettings.cs

```diff
@@ -0,0 +1,85 @@
+#if RECORDER_AVAILABLE
+using System;
+using System.Collections.Generic;
+using pxr;
+using Unity.Formats.USD;
+using UnityEditor.Recorder;
+using UnityEngine;
+
+namespace UnityEditor.Formats.USD.Recorder
+{
+    [RecorderSettings(typeof(UsdRecorder), "Usd Clip", "usd_recorder")]
```

Comment:
I think it should be "**USD** Clip" (with all caps). I'm not sure if it's the convention but it appears to be capitalized in the [official Pixar documentation](https://graphics.pixar.com/usd/docs/index.html).